src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java

```diff
@@ -68,6 +69,8 @@ private static PartyDetails buildPartyDetails(ApplicantDto applicantDto) {
                 .canYouProvidePhoneNumber(applicantDto.getApplicantContactDetail().getCanProvideTelephoneNumber())
                 .email(isNotEmpty(applicantDto.getApplicantContactDetail().getEmailAddress())
                         ? applicantDto.getApplicantContactDetail().getEmailAddress() : null)
+                .contactPreferences(ContactPreferences.fromValue(
+                    applicantDto.getApplicantContactDetail().getApplicantContactPreferences()))
```

Comment:
line 73, any chance of encountering null pointer of applicantDto.getApplicantContactDetail()

Reply:
Contact details are mandatory fields in the UI, it can never be null.
That's why there is no null check in above lines as well.

Reply:
thanks patil. with the above understanding marking it as resolved.